src/app/ats-guard.guard.ts

```diff
@@ -12,23 +11,17 @@ export class AtsPageGuard {
   constructor(private navService: NavService,
     private authService: AuthService,
     private cookieService: CookieService,
-    private authAccessService: AuthAccessService,
   ) { }
 
   canActivate(): any {
-    if (this.navService.isHris == false) {
+    if (false == this.navService.isHris) {
```

Comment:
What's the point of this switch over?

Reply:
If someone accidently removes the equal sign then it becomes a variable assignment and not a comparison. This bug will be difficult to find. However I do note that Angular uses "===" for assignments and if a dev. removes an equal then the compiler does detect it. Unlike in c#.